apps/web/src/components/Settings/Delete/index.tsx

```diff
@@ -120,7 +120,7 @@ const DeleteSettings: FC = () => {
           </div>
           <p>
             <Trans>
-              This will permanently delete your Profile NFT on the Lens
+              This will permanently burn your Profile NFT on the Lens
```

Comment:
Burn doesn't makes sense for normal users, so I'll leave it to 'delete'

Reply:
Kk i have changed it back